src/test/java/org/folio/rest/domain/AccountPaymentStatusTest.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.rest.domain;
+
+import static org.folio.rest.jaxrs.model.PaymentStatus.Name.fromValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.folio.rest.jaxrs.model.Account;
+import org.folio.rest.jaxrs.model.PaymentStatus;
+import org.folio.test.support.EntityBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class AccountPaymentStatusTest {
+
```

Comment:
If `fromValue()` is the method that throws `IllegalArgumentException` when invalid `PaymentStatus` is supplied, then calling `fromValue` **only** should be sufficient to test both cases - no need to build the Account.

Reply:
fixed